### PR TITLE
Replace class names with ::class references

### PR DESCRIPTION
**Pull request by nightlinus**
PR for issue https://github.com/zendframework/zend-expressive/issues/245

### CODE REVIEW

File: test/AppFactoryTest.php [hunk 1]
```diff
@@ -29,36 +29,36 @@ public function getRouterFromApplication(Application $app)
     public function testFactoryReturnsApplicationInstance()
     {
         $app = AppFactory::create();
-        $this->assertInstanceOf('Zend\Expressive\Application', $app);
+        $this->assertInstanceOf(\Zend\Expressive\Application::class, $app);
```
Comment (samsonasik) [inline test/AppFactoryTest.php]: `Application::class` is enough

**Comment by nightlinus** [comment]
Do you mean to simlify all FQN's?

**Comment by weierophinney** [comment]
Yes. It they're imported, there's no reason to fully qualify them.

**Comment by weierophinney** [comment]
My only note is that any class you're referencing via `::class` notation should likely be imported. I can take care of that during merge, however.

Thanks, @nightlinus !

**Pull request merged**